.expeditor/config.yml

```diff
@@ -23,6 +23,9 @@ github:
   # The file where our CHANGELOG is kept. This file is updated automatically with
   # details from the Pull Request via the `built_in:update_changelog` merge_action.
   changelog_file: "CHANGELOG.md"
+  # This will trigger an Omnibus build with EXPIRE_CACHE enabled after
```

Comment:
Intentional?

Reply:
Yup; the pipelines are sadly fragile when changing ruby versions and this is the safest way to get working chef-12 builds.